build.yaml

```diff
@@ -2455,3 +2455,7 @@ node_modules:
   - src/node/ext/server.cc
   - src/node/ext/server_credentials.cc
   - src/node/ext/timeval.cc
+python_dependencies:
```

Comment:
Consider modifying tools/buildgen/build-cleaner.py to lift this section towards the top.